misc/move_migrations_to_sqitch.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
```

Comment:
This file looks like something you ran to create the new files - does it need to be kept around?

Reply:
It doesn't, I'll kill it.

Reply:
killed.